backend/mongodb_handler.py

```diff
@@ -0,0 +1,249 @@
+from datetime import datetime
+from typing import Any, Dict, List, Optional, cast
+
+from bson.objectid import ObjectId
+from pymongo import MongoClient
+
+from .document_status import Status
+from .role import Role
+from .validators import role_validation
+
+
+class MongoDBHandler:
+    def __init__(self) -> None:
+        self.client = MongoClient(port=27017)
+        self.db = self.client.myDatabase
+
+    def create_user(
+        self, username: str, user_role: str, company: str
+    ) -> Optional[ObjectId]:
+        if (
+            role_validation(user_role)
```

Comment:
Validation should be in view layer, not in model layer. If role is wrong, then response will contains 4** error code and some message